tsdb/blockwriter.go

```diff
@@ -90,16 +89,10 @@ func (w *BlockWriter) Appender(ctx context.Context) storage.Appender {
 // Flush implements the Writer interface. This is where actual block writing
 // happens. After flush completes, no writes can be done.
 func (w *BlockWriter) Flush(ctx context.Context) (ulid.ULID, error) {
-	seriesCount := w.head.NumSeries()
```

Comment:
I was only trying to remove line 94-96 because if I backfill a recording rule and there are no series created for that rule, it errors on line 95. Plus when I flush when there are no series, it does not create a block so I'm not sure why the  `w.head.NumSeries() == 0 ` check was being made.

However I can keep line 93 and line 102 to log that info.